pkg/logs/logs.go

```diff
@@ -53,6 +94,18 @@ func (o *LogsClient) DevModeLogs(componentName string, namespace string) ([]map[
 		}
 	}
 
+	if !ignorePods {
+		// pods in Deploy mode get ignored by matchOwnerReferenceWithResources function since they are not assigned
+		// a default owner reference like a pod from Dev mode
+		for _, resource := range resources {
+			for _, pod := range podList.Items {
+				if pod.GetUID() == resource.GetUID() {
+					pods = append(pods, pod)
+				}
+			}
+		}
```

Comment:
Do we need to loop over pods and resources?

I think that getting all the Pods resources from the `resources` would be sufficient

Reply:
`resources` contains all the resources on the cluster that are matching the selector in the given namespace. `podList` is all the Pods in the namespace, irrespective of whether they match the selector or not.